Assets/_nathan/materials.meta

```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
```

Comment:
This looks broken. I will remove all meta files in .gitignore which should solve the problem.

Reply:
do I need to make any changes?